src/plugins/irc-events/message.js

```diff
@@ -91,6 +91,9 @@ module.exports = function(irc, network) {
 		});
 		chan.pushMessage(client, msg, !self);
 
-		LinkPrefetch(client, chan, msg);
+		// No prefetch URLs unless are simple messages.
+		if (data.type === Msg.Type.MESSAGE) {
```

Comment:
@xPaw, I'd rather not. I'd prefer a whitelist than a blacklist, at least at this stage and for v2.2.
That would mean filtering out topics as well probably, parts, quits, aways, etc.